src/DataFixtures/AppFixtures.php

```diff
@@ -113,9 +124,9 @@ public function load(ObjectManager $manager): void
                         $worklog->setDescription("Beskrivelse af worklog-$key-$i-$j-$k");
                         $worklog->setIsBilled(false);
                         $worklog->setProject($project);
-                        $worklog->setWorker('test@test');
+                        $worklog->setWorker($workerArray[rand(0, 9)]);
```

Comment:
I think it is better to make fixtures stable instead of random.
This makes it easier to write tests.